editor/plugins/shader_editor_plugin.cpp

```diff
@@ -39,14 +39,18 @@
 #include "editor/editor_scale.h"
 #include "editor/editor_settings.h"
 #include "editor/filesystem_dock.h"
-#include "editor/plugins/visual_shader_editor_plugin.h"
 #include "editor/project_settings_editor.h"
 #include "editor/shader_create_dialog.h"
+#include "modules/modules_enabled.gen.h"
 #include "scene/gui/split_container.h"
 #include "servers/display_server.h"
 #include "servers/rendering/shader_preprocessor.h"
 #include "servers/rendering/shader_types.h"
 
+#ifdef MODULE_VISUAL_SHADER_ENABLED
+#include "modules/visual_shader/editor/visual_shader_editor_plugin.h"
+#endif // MODULE_VISUAL_SHADER_ENABLED
```

Comment:
For the cases where module includes are needed, it should be done like this:
```
#include "modules/modules_enabled.gen.h" // For visual shader.
#ifdef MODULE_VISUAL_SHADER_ENABLED
#include "modules/visual_shader/editor/visual_shader_editor_plugin.h"
#endif
```
as one block after the regular includes.